geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractHitsMissesIntegrationTest.java

```diff
@@ -322,6 +322,13 @@ public void testPersist() {
   }
 
   /************* Sorted Set related commands *************/
+  @Test
+  public void testZAdd() {
+    Map<String, Double> toAdd = new HashMap<>();
+    toAdd.put("member", 1.0);
+    runCommandAndAssertNoStatUpdates("key", (String k) -> jedis.zadd(k, toAdd));
+  }
+
```

Comment:
There's actually already a `SortedSet related commands` section above this one that has another ZAdd hits/misses test. I think we just need to move the `testZIncrBy` test below this into that section.

Reply:
I'm so tired of doing rebases. this is the kind of stuff that git always messes up

Reply:
I'm sorry 😞 Thank you for doing it, though!